core/config/value.go

```diff
@@ -421,6 +423,17 @@ func (cv Value) getValueStruct(key string, target interface{}) (interface{}, err
 				continue
 			}
 
+			if fieldType == reflect.TypeOf(_zeroTimeDuration) {
```

Comment:
I totally agree with being aware of writing perf critical code in hot paths. #willnothappenagain